src/main/scala/cromwell/engine/CallActorState.scala

```diff
@@ -0,0 +1,35 @@
+package cromwell.engine
+
+sealed trait CallActorState {
+  def isTerminal: Boolean
+}
+
+case object CallNotStarted extends CallActorState {
```

Comment:
No need to make objects final

Since CallActor is now quite simple, having everything together and self-contained probably makes sense, I'll shuffle it in.